scipy/sparse/linalg/_isolve/iterative.py

```diff
@@ -3,15 +3,16 @@
 from scipy.sparse.linalg._interface import LinearOperator
 from .utils import make_system
 from scipy.linalg import get_lapack_funcs
+from scipy._lib.deprecation import _NoValue
 
 __all__ = ['bicg', 'bicgstab', 'cg', 'cgs', 'gmres', 'qmr']
 
 
-def _get_atol(name, b, tol=None, atol=0., rtol=1e-5):
+def _get_atol(name, b, tol=_NoValue, atol=0., rtol=1e-5):
     """
     A helper function to handle tolerance deprecations and normalization
     """
-    if tol is not None:
+    if tol is not _NoValue:
         msg = (f"'scipy.sparse.linalg.{name}' keyword argument 'tol' is "
                "deprecated in favor of 'rtol' and will be removed in SciPy "
                "v.1.14.0. Until then, if set, will override 'rtol'.")
```

Comment:
Suggestion for drive-by typo fix
```suggestion
               "v.1.14.0. Until then, if set, it will override 'rtol'.")
```